package.json

```diff
@@ -24,6 +24,7 @@
     "@turf/turf": "^6.5.0",
     "@types/leaflet": "^1.9.3",
     "leaflet": "^1.9.3",
+    "lodash": "^4.17.21",
```

Comment:
What exactly are we using lodash for?

Reply:
I'm a lodash addict. I used a few in a few places... `_without()` (to find and remove a provided element from an array), and `_isEqual()` (a deep array comparison).